ovos_utils/gui.py

```diff
@@ -795,8 +795,8 @@ def _normalize_page_name(page_name: str) -> str:
         file, ext = splitext(page_name)
         if ext == ".qml":
             log_deprecation("GUI resources should exclude gui-specific file "
-                            "extensions. In the future, this may cause "
-                            "unexpected behavior.", "0.1.0")
+                            f"extensions. This call should probably pass "
+                            f"`{file}`, instead of `{page_name}`", "0.1.0")
```

Comment:
reversed, pass page_name instead of file

Reply:
in this context `file` is the basename and `page_name` is what was passed to the function